x/cellarfees/keeper/abci_test.go

```diff
@@ -2,13 +2,17 @@ package keeper
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/golang/mock/gomock"
+	gravitytypes "github.com/peggyjv/gravity-bridge/module/v4/x/gravity/types"
 	appParams "github.com/peggyjv/sommelier/v7/app/params"
+	auctionTypes "github.com/peggyjv/sommelier/v7/x/auction/types"
 	cellarfeesTypes "github.com/peggyjv/sommelier/v7/x/cellarfees/types"
 )
 
 func (suite *KeeperTestSuite) TestBeginBlockerZeroRewardsBalance() {
 	ctx, cellarfeesKeeper := suite.ctx, suite.cellarfeesKeeper
+
```

Comment:
I think you need to add the same fee accrual initialization here, the test panics